src/lambda/oembed/oembed.js

```diff
@@ -34,7 +34,8 @@ function handler(event, context, callback) {
     );
   }
 
-  const { url, referrer, maxwidth = 900, maxheight = 300 } = params;
+  const { url, referrer, maxwidth = '100%', maxheight = 300 } = params;
+  const iframeSrc = url.replace('gist', 'embed');
```

Comment:
Should we also provide a default set of panes? All 4 panes on 900 pixels is a bit much?

Reply:
I'll add `preview` & `result`, the defaults that are used in the GUI

Reply:
Not entirely sure about this though. The user can customize the url / embed code via the embed frame if be wants to. And it's hard to decide what are the right frames to show, as that can depend on the situation.

If we choose a default, I vote for `markup,preview,query`. What do you think?

Reply:
Then I think we should make that the default in the GUI too to be consistent

Reply:
Ok. Let's merge this, and look at that another time 🙂